src/ledger/LedgerTxn.cpp

```diff
@@ -2089,6 +2089,12 @@ void
 LedgerTxn::Impl::updateEntryIfRecorded(InternalLedgerKey const& key,
```

Comment:
While you're changing this, I think that `void LedgerTxn::Impl::updateEntry(InternalLedgerKey const&, EntryMap::iterator const*, LedgerEntryPtr)` is unused and can be removed.

Reply:
Done